src/main/java/com/github/fishio/SinglePlayerPlayingField.java

```diff
@@ -69,6 +69,8 @@ public void postTick() {
 				}
 			}
 		});
+		
+		new PowerUpSpawner(this);
```

Comment:
Perhaps it would be useful to store a reference to the PowerUpSpawner, so it can be interacted with later?

<naming